src/Kdyby/Doctrine/Console/CommandHelper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * This file is part of the Kdyby (http://www.kdyby.org)
+ *
+ * Copyright (c) 2008 Filip Procházka (filip@prochazka.su)
+ *
+ * For the full copyright and license information, please view the file license.txt that was distributed with this source code.
+ */
+
+namespace Kdyby\Doctrine\Console;
+
+use Doctrine\DBAL\Tools\Console\Helper\ConnectionHelper;
+use Doctrine\ORM\Tools\Console\Helper\EntityManagerHelper;
+use Kdyby\Console\ContainerHelper;
+use Nette;
+
+
+
+/**
+ * @author Tomáš Jacík <tomas@jacik.cz>
+ */
+abstract class CommandHelper extends Nette\Object
```

Comment:
Why abstract? :) if you wanned to create a static class, it's better to make it final and add a private construtor to it.